src/converse-muc-views.js

```diff
@@ -841,7 +841,8 @@ converse.plugins.add('converse-muc-views', {
                     );
                     return false;
                 }
-                if (!this.model.occupants.findWhere({'nick': args[0]}) && !this.model.occupants.findWhere({'jid': args[0]})) {
+                if (!(_.includes(['admin', 'ban', 'owner', 'member', 'revoke'], command) && u.isValidJID(args[0])) &&
```

Comment:
Please create a `const AFFILIATION_CHANGE_COMANDS` at the top of the file which is set to these commands and then use that here.

Would be nice if you also wrapped the line.

And then please rebase again, so that I can rebase from github.

Thanks.

Reply:
@jcbrand Done.